impress-org/givewp
# Feature: add excerpt for v3 forms

Resolves [GIVE-165]

## Description



This PR added a new "Excerpt" text area option in the V3 general form settings to be used to add a summary to the forms. This way, we can fix a problem in the Form Grid shortcode/block/widget that has this value missing for V3 forms due to the missing of this option. Also, this PR adds a migration step to transfer the "Excerpt" option from V2 to V3 forms.

## Affects



- The V3 form general settings
- The form grid shortcode/block/widget
- The migration form process

## Visuals



https://www.loom.com/share/25692c76f0ce4e04bf55c12bb0edc19c?sid=21d46133-64bd-4d9a-a9c4-09f4a5b6df72

![image](https://github.com/impress-org/givewp/assets/16308864/37c9c4c1-5a83-4ad7-88ae-7b3a029e4587)

## Testing Instructions



1. Create or edit a V3 form and fill the new Excerpt text area in the general settings;
2. Add the Form Grid widget to an Elementor page;
3. Set the "Show Excerpt" to "true";
4. The "Excerpt" now should be visible in the form grid.

## Pre-review Checklist






[GIVE-165]: https://stellarwp.atlassian.net/browse/GIVE-165?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
